packages/create-sitecore-jss/src/templates/react/package.json

```
@@ -0,0 +1,122 @@
+{
+  "name": "<%- appName %>",
+  "description": "Application utilizing Sitecore JavaScript Services and React (create-react-app).",
+  "version": "20.0.0-canary",
+  "private": true,
+  "config": {
+    "appName": "<%- appName %>",
+    "rootPlaceholders": [
+      "<%- appPrefix ? `${helper.getPascalCaseName(appName)}-` : '' %>jss-main"
```

```suggestion
      "<%- helper.getAppPrefix(appPrefix, appName)%>jss-main"
```
use `getAppPrefix` helper here